python/lsst/ts/eas/eas_csc.py

```diff
@@ -33,11 +33,14 @@
 from .config_schema import CONFIG_SCHEMA
 
 SAL_TIMEOUT = 5.0  # SAL telemetry/command timeout
+M1M3TS_STOP_TIMEOUT = 30.0  # Time to wait for fans and valve to stop
```

Comment:
should be fans to stop and valve to close

Also we might need a bit more time. It takes probably 60 seconds, maybe more to stop fans from the max speed - we can try that today